doc/tutorials/visualizing_in_rviz/visualizing_in_rviz.rst

```diff
@@ -116,7 +126,7 @@ After we've constructed and initialized, we now create some closures (function o
     auto const draw_title = [&moveit_visual_tools](auto text) {
       auto const text_pose = [] {
         auto msg = Eigen::Isometry3d::Identity();
-        msg.translation().z() = 1.0;
+        msg.translation().z() = 1.0; //Place text 1m above the base link
```

Comment:
```suggestion
        msg.translation().z() = 1.0;  // Place text 1m above the base link
```
Our clang-format enforces 2 spaces before trailing comments, but I suppose it doesn't affect these rst files 😂